### DISCUSSION TITLE
Alternate Pin For SDI(MOSI)=23

### DISCUSSION THREAD

**Discussion by Sudupatil** [Bodmer/TFT_eSPI#2099] [Compatible support libraries]
Hello,

I am working on one project where i used HX711_ADC library, For that I used Pin 23 as HX711_sck, which i think conflict with your library.  

So i would like to ask you can i use any other Pin From ESP32 Wroom to MOSI?  

Display ILI9488 And Pin configuration are as 
T_IRQ	-
T_DO	19
T_DIN	23
T_CS	13
T_CLK	18

SCK	           18
SDI(MOSI)   23
DC	           4
RESET	   2
CS	          15

Thanks and regards
Sudarshan Patil

**Comment by Bodmer** [comment]
Yes, you can allocate another pin in the setup file.

FYI: The ESP32 contains a multiplexer so the MOSI signal can be routed to other pins. The caveate is that using the multiplexer means the maximum recommended SPI clock rate reduces from 80MHz to 40MHz, however that will not affect your setup as the ILI9488 cannot run at such high speed anyway.

**Reply by Sudupatil** [reply]
Hello,

Thank you for your timely reply, I got Reply from Owner of HX711_ADC library and he/she suggested some other pins and they worked. 

so i am comfortable with your original pin configuration. and i did not need to change it. Having said that i had observed some issues with ILI9488 Display, I will test it again, it perhaps because conflict in PIN. if issue, i will contact you .

Thanks and regards
Sudarshan Patil
